app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientsViewController.kt

```diff
@@ -89,16 +91,23 @@ class RecentPatientsViewController @Inject constructor(
         recentPatients
       }
 
-  private fun recentPatientItem(recentPatient: RecentPatient) =
-      RecentPatientItem(
-          uuid = recentPatient.uuid,
-          name = recentPatient.fullName,
-          age = age(recentPatient),
-          gender = recentPatient.gender,
-          updatedAt = recentPatient.updatedAt,
-          dateFormatter = dateFormatter,
-          clock = userClock
-      )
+  private fun recentPatientItem(recentPatient: RecentPatient): RecentPatientItem {
+    val today = Instant.now(userClock).toLocalDateAtZone(userClock.zone)
```

Comment:
What do you think of passing the current date to this method as a parameter since it should be the same for all the recent patients? We can compute this once and pass it.

Reply:
Done